css/css-display/display-math-on-non-mathml-elements.html

```diff
@@ -8,14 +8,18 @@
 <div id="container">
 
   <div data-expected="block" style="display: math;"></div>
-  <div data-expected="inline" style="display: inline-math;"></div>
+  <div data-expected="block" style="display: inline-math;"></div>
```

Comment:
This looks real confusing, since it's actually just testing that `inline-math` is an invalid keyword. Moving it to a separate section, with a note about what it's doing, would prevent future confusion. ^_^